tools/CouchDB_Import_Instruments.php

```diff
@@ -207,7 +211,7 @@ function updateCandidateDocs($Instruments)
             $JSONData      = $instrumentObj->usesJSONData();
             $tableName     = "";
             if ($JSONData === false) {
-                $tableName = $instrumentObj->table;
+                $tableName = $instrumentObj->table ?? "";
```

Comment:
Fixes an error when a LINST file doesn't have a `.meta` file present. Without this "tableName" is null instead of the empty string and causes an error when used

Reply:
It looks like there's no `table` tag in the bmi linst file that comes with raisinbread.

https://github.com/aces/Loris/blob/main/raisinbread/instruments/bmi.linst

(This seems like it was always the case since the reason I encountered the bug was because I had copied bmi.linst into my project/instruments directory before the creation of the .meta file..)

Reply:
no, it was only removed when the meta was added
https://github.com/aces/Loris/commit/544da8d62f9948cf29226d4540baccd4f669b925#diff-3047eea24c5234ad4857c3257d698a9efb3af98136856bc04dfbc5484f0b1c2c

Reply:
ok, that's weird.. I had a bmi.linst with no table tag but no meta file either on my sandbox. Should I just revert this part?

Reply:
I don't think theres harm in it, but I just wanted to make sure its not a core issue...